docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

```diff
@@ -1009,6 +1014,9 @@ template, and use it instead of the default key when present.
 
 All old contracts will default to using the ``issuer``, and new contracts
 will use the ``alternativeKey`` field.
+Note also that key expressions in upgrabable packages cannot include types
+from other packages, with the exception of `daml-stdlib` and `daml-prim`.
+See `Limitations <#limitatiions>`__ for more information.
```

Comment:
@dylant-da @remyhaemmerle-da @paulbrauner-da let me know what you think.

Reply:
LGTM